Base/QTCore/qSlicerExtensionsManagerModel.cxx

```diff
@@ -243,6 +243,7 @@ class qSlicerExtensionsManagerModelPrivate
   QNetworkAccessManager NetworkManager;
 
   qRestAPI ExtensionsMetadataFromServerAPI;
+  bool UseOffset = true;
```

Comment:
useOffset

Reply:
Since all the other internal variable starts with an upper case letter, I suggest to stay consistent.